tests/python/unittest/test_meta_schedule_task_scheduler.py

```diff
@@ -140,6 +140,9 @@ def __init__(self):
         self.records = []
         self.workload_reg = []
 
+    def has_workload(self, mod: IRModule) -> bool:
+        return False
```

Comment:
Should we query `workload_reg` here instead of `return False` simply?

Reply:
Aha this is a dummy database for mock testing